sabnzbd/scheduler.py

```diff
@@ -383,6 +383,12 @@ def __check_diskspace(self, full_dir: str, required_space: float):
         if not sabnzbd.Downloader.paused:
             self.cancel_resume_task()
 
+    def __resume_downloading(self):
+        """Unpause Downloader if paused"""
+        if sabnzbd.Downloader.paused:
```

Comment:
Seems the check for it being paused is redundant? The `Downloader.resume()` also performs this check.